plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestParquetPageSkipping.java

```diff
@@ -88,6 +88,27 @@ public void testAndPredicates()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testRandomData()
+    {
+        String tableName = "test_random_" + randomTableSuffix();
+        int updateCount = 8192;
+        assertUpdate(
+                "CREATE TABLE " + tableName + " (col) WITH (format = 'PARQUET') AS " +
+                        "SELECT * FROM unnest(transform(repeat(1, 8192), x -> rand()))",
+                updateCount);
+        for (int i = 0; i < 8; i++) {
+            assertUpdate(
+                    "INSERT INTO " + tableName + " SELECT rand() FROM " + tableName,
+                    updateCount);
+            updateCount += updateCount;
+        }
+        for (double i = 0; i < 1; i += 0.1) {
+            assertSameResults(format("SELECT * FROM %s WHERE col BETWEEN %f AND %f", tableName, i - 0.0001, i + 0.0001));
```

Comment:
`assertSameResults` is not meaningful name, since it doesn't tell "same as .." ?

let's call it `assertColumnIndexResults`

Reply:
added a prep commit which renames this